src/index.ts

```diff
@@ -306,11 +268,15 @@ export class IDX {
   }
 
   async _createContent(
-    definition: Definition,
+    definition: DefinitionWithID,
     content: unknown,
     options?: CreateOptions
   ): Promise<DocID> {
-    const doc = await this._createDocument(content, { schema: definition.schema }, options)
+    const doc = await this._createDocument(
+      content,
+      { family: definition.id.toString(), schema: definition.schema },
+      options
+    )
```

Comment:
Should we be setting `deterministic: true` here? Seems to be missing.
Also schema should also not be set here, but on first update.